Source/DafnyCore/Resolver.cs

```diff
@@ -14875,7 +14874,11 @@ public ResolutionContext(ICodeContext codeContext, bool isTwoState)
         if (currentClass is ClassDecl cd && cd.IsDefaultClass) {
```

Comment:
It looks like this case would also hit a crash, just later on in the pipeline?

Reply:
How would you reproduce it?

Reply:
I didn't have any luck, and it looks like the error immediately above should always be triggered, which seems to prevent any further problems.